randovania/games/am2r/json_data/The Tower.txt

```diff
@@ -153,19 +153,23 @@ Extra - map_name: rm_a4h03
   * Layers: default
   * Open Passage to Tower Exterior North East/Dock to Tower Exterior North
   > Top
-      Trivial
+      Space Jump or Walljump (Intermediate) or Can Use Spider Ball
 
 > Dock to Tower Exterior West; Heals? False
   * Layers: default
   * Open Passage to Tower Exterior West/Dock to Tower Exterior North
   > Top
-      Trivial
+      Space Jump or Walljump (Intermediate) or Can Use Spider Ball
 
 > Dock to Dark Maze (Left); Heals? False
   * Layers: default
   * Open Passage to Dark Maze/Dock to Tower Exterior North (Left)
   > Top
-      Trivial
+      All of the following:
+          Any of the following:
+              Enabled Softlock Prevention
+              Mid-Air Morph (Beginner) and Can Use Any Bombs
+          Space Jump or Walljump (Intermediate) or Can Use Spider Ball
```

Comment:
HJ + beginner WJ probably too. same with the connection in dark maze
Also CBJ probably too